CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## 2.26.0-rc.0 / 2021-03-24
+
+Notable changes:
+
+* Prometheus UI helps you to write correct PromQL queries, with auto-completion, syntax highlightning and linting. 
+* Users replicating metrics via remote_write to AWS backends does not require extra sidecar, Prometheus now supports that method natively.
+* If you enable `--enable-feature=exemplar-storage`, Prometheus will hold certain amount of scraped OpenMetrics' Exemplars in circular memory. It will also expose `/api/v1/query_exemplars` querying HTTP API.
+* If you enable `--enable-feature=promql-negative-offset`, PromQL will support negative offset in queries eg. `sum_over_time(metric[1h:1m] offset -5m)`.
```

Comment:
Might be useful to link to the docs here. Also, it's a comma separated list, the way this is written reads a bit like both can be flags on their own, which might confuse people, but not sure there's a better way to write it.

```suggestion
* Two experimental features have been added, these are [disabled by default and may change in the future](https://prometheus.io/docs/prometheus/latest/disabled_features/):
    * If you enable `--enable-feature=exemplar-storage`, Prometheus will hold certain amount of scraped OpenMetrics' Exemplars in circular memory. It will also expose `/api/v1/query_exemplars` querying HTTP API.
    * If you enable `--enable-feature=promql-negative-offset`, PromQL will support negative offset in queries eg. `sum_over_time(metric[1h:1m] offset -5m)`.
```

Reply:
Oh multiple flags do work, nevermind...